### PR TITLE
add cassandra sink doc

### LINKED ISSUE #1275: Document: feat(sink): Support cassandra sink
This issue tracks the documentation update needed for the merged PR #11878.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/11878
Source PR Merged At: 2023-09-07T14:02:43Z

### PR DESCRIPTION
**Pull request by hengm3467**
## Info

- **Description**

  Add a guide about sinking data to Cassandra / ScyllaDB.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/11878

- **Related doc issue**
  
  Resolves #1275

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/guides/sink-to-cassandra.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+The Cassandra sink connector in RisingWave is currently an experimental feature. Its functionality is subject to change. We cannot guarantee its continued support in future releases, and it may be discontinued without notice. You may use this feature at your own risk.
+:::
+
+## Prerequisites
+
+- Ensure your Cassandra or ScyllaDB cluster is accessible from RisingWave.
+
+- The Cassandra sink connector in RisingWave relies on the connector node to work. Please ensure the connector node is enabled in RisingWave. For details, see [Enable the connector node](/deploy/risingwave-trial.md/?method=binaries#optional-enable-the-connector-node).
+
+## Create a Cassandra sink
```
Comment (emile-00) [inline docs/guides/sink-to-cassandra.md]: Section title should be Syntax to keep consistent with the other guides

File: docs/guides/sink-to-cassandra.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+| Parameter Names       | Description |
+| --------------------- | ---------------------------------------------------------------------- |
+|*sink_name*| Name of the sink to be created.|
+|*sink_from*| A clause that specifies the direct source from which data will be output. *sink_from* can be a materialized view or a table. Either this clause or *select_query* query must be specified.|
+|AS *select_query*| A `SELECT` query that specifies the data to be output to the sink. Either this query or a *sink_from* clause must be specified. See [SELECT](/sql/commands/sql-select.md) for the syntax and examples of the `SELECT` command.|
+| `type`                | Required. Specify if the sink should be `upsert` or `append-only`. If creating an `upsert` sink, you must specify a primary key.|
+| `primary_key`          | Optional. A string of a list of column names, separated by commas, that specifies the primary key of the Cassandra sink.|
+|`force_append_only`| If `true`, forces the sink to be `append-only`, even if it cannot be.|
+| `cassandra.url`        | Required.The URL or IP address of the Cassandra or ScyllaDB cluster or node you want to connect to.|
```
Comment (emile-00) [inline docs/guides/sink-to-cassandra.md]: Missing a space after the first period

**Review by emile-00 — COMMENTED** [review]
Guide missing from sidebar

**Comment by neverchanje** [comment]
Thanks. LGTM

**Review by neverchanje — APPROVED** [review]

**Pull request merged**